guides/common/modules/con_supported-usage-of-project-components.adoc

```diff
@@ -13,11 +13,6 @@ Reserve this instance solely for the purpose of running {ProjectName}.
 Not supported: Using the operating system included with {Project} to run other daemons, applications, or services within your environment.
 endif::[]
 
-SELinux::
```

Comment:
I don't think we should be mentioning this here (Planning guide) because we already list it in the system prerequisites (Installation guide).

Reply:
I agree with dropping it: It's a requirement but not purely a Foreman component (even though there's foreman-selinux.rpm).